text/tests/test_check_style.py

```diff
@@ -0,0 +1,19 @@
+from subprocess import Popen, PIPE
+import logging
+
+
+def test_check_style():
```

Comment:
are we already enabled style check? it's very good to have it.

Reply:
I would suggest not to add this in this PR because so many other things are changing.